druid-shell/src/platform/x11/window.rs

```diff
@@ -1117,21 +1151,56 @@ fn key_mods(mods: u16) -> Modifiers {
     ret
 }
 
+/// A handle that can get used to schedule an idle handler. Note that
+/// this handle is thread safe.
```

Comment:
Why does the handle have to be thread safe? (See my other comment about threads)